### PR TITLE
Fineract 2081: Enhance technical documentation

### PR DESCRIPTION
**Pull request by Napho**
## Description

Describe the changes made and why they were made.

Descriptions have been added to property file value documentation in fineract-doc module

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [x] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-doc/src/docs/en/chapters/appendix/properties-resilience4j.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
+= Resilience4j Properties
+
+For a deeper understanding of reslience4j, refer to the https://resilience4j.readme.io/docs/getting-started[Official website]
```
Comment (kjozsa) [inline fineract-doc/src/docs/en/chapters/appendix/properties-resilience4j.adoc]: please fix the typo here

Comment (Napho) [reply fineract-doc/src/docs/en/chapters/appendix/properties-resilience4j.adoc]: @kjozsa this has been corrected.

**Review by kjozsa — APPROVED** [review]
LGTM

**Comment by kjozsa** [comment]
@Napho please also filll the ticket description as expected. Thank you

**Comment by adamsaghy** [comment]
@Napho Please amend the commit message to fit for the required format:
"FINERACT-2981: Enhance technical documentation"

After the Fineract jira ticket there should be a colon and after the short description.

The actual description is misleading: "FINERACT-2081 add descriptions to property file values"

**Comment by Napho** [comment]
> @Napho Please amend the commit message to fit for the required format: "[FINERACT-2981](https://issues.apache.org/jira/browse/FINERACT-2981): Enhance technical documentation"
> 
> After the Fineract jira ticket there should be a colon and after the short description.
> 
> The actual description is misleading: "[FINERACT-2081](https://issues.apache.org/jira/browse/FINERACT-2081) add descriptions to property file values"

@adamsaghy thanks for that, ive updated it accordingly

**Comment by kjozsa** [comment]
> > @Napho Please amend the commit message to fit for the required format: "[FINERACT-2981](https://issues.apache.org/jira/browse/FINERACT-2981): Enhance technical documentation"
> > After the Fineract jira ticket there should be a colon and after the short description.
> > The actual description is misleading: "[FINERACT-2081](https://issues.apache.org/jira/browse/FINERACT-2081) add descriptions to property file values"
> 
> @adamsaghy thanks for that, ive updated it accordingly

sorry @Napho, but the correct commit message format is `FINERACT-xxx: <description>`. Please be exact, even with upper/lowercase letters!

**Pull request merged**